LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 sevonj
```

Comment:
I _can_ take all the credit, but how about "Ryhmä 4"

Reply:
Yeah good point :DD updated it